classes/category/Repository.php

```diff
@@ -62,6 +63,34 @@ public function exists(int $id, int $contextId = null): bool
         return $this->dao->exists($id, $contextId);
     }
 
+    /**
+     * Get the breadcrumb of a category
+     *
+     * @return string For example: Social Sciences > Anthropology
+     */
+    public function getBreadcrumb(Category $category, ?Category $parent = null): string
+    {
+        return !$parent
```

Comment:
Makes sense. I've implemented it the same as how we use the `common.commaListSeparator`, which still enforces a particular order but let's them change the character:

```po
msgid "common.breadcrumbSeparator"
msgstr " > "
```

```php
$parent->getLocalizedTitle()
    . __('common.breadcrumbSeparator')
    . $category->getLocalizedTitle();
```

Is that alright or do you want me to include the names in the full locale string?

Reply:
I would rather a locale key that includes both variables in the locale string, like:
```
msgid "common.breadcrumb"
msgstr "{$parentBreadcrumb} > {$childBreadcrumb}
```
This makes it clearer to translators, and makes it less likely that this will get inappropriately reused.